pilot/pkg/security/model/log.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	istiolog "istio.io/pkg/log"
+)
+
+var log = istiolog.RegisterScope("authn", "authn", 0)
```

Comment:
I feel it might be better to use an explicit name, like `authnLog`

Reply:
While descriptive names are nice, the pattern throughout pilot seems to be to have a log var named 'log'.  I think if we are going to change this pattern, we should do this in the scope of its own issue and PR...